### DISCUSSION TITLE
No access to cmark-gfm processor

### DISCUSSION THREAD

**Discussion by ThibauldMichel** [KDE/ghostwriter#647] [Q&A]
As the documentation mention, my different words processor have been identified by Ghostwriter and are accessible in the "export" menu.

However cmark-gfm is not accessible in the menu.

Does anyone know if it is an issue or a missing feature?

**Comment by wereturtle** [comment]
With ghostwriter 2, cmark-gfm is the default, built-in processor.  The option should be the top-most one when exporting.

![image](https://user-images.githubusercontent.com/10375170/130333466-4002a1ca-1228-48fa-9121-ccb2c2fc9465.png)
